styleguide/components/31-molecules/fields/checkbox-with-filter/checkbox_filter.functions.js

```diff
@@ -204,7 +207,7 @@
     const makeTag = (checkbox, label) => {
       let tag = document.createElement('span');
       tag.className = 'tag filter';
-      tag.innerText = label.innerText;
+      tag.textContent = label.textContent;
```

Comment:
I'm curious why this changed, looks interesting.

Reply:
I included it in the description of the PR 😆 So:

> Replaced innerText by textContent, since innerText doesn't return anything if the element has display: none;

i.e. when the page loads and there are pre-selected checkboxes (and the popup isn't displayed), the tags in the popup were empty because `innerText` doesn't return anything if the popup had `display: none;`